cmd/yggd/grpc.go

```diff
@@ -34,9 +34,10 @@ type dispatcher struct {
 	workers     map[string]worker
 	pidHandlers map[int]string
 	httpClient  *http.Client
+	config      *pb.Config
 }
 
-func newDispatcher(httpClient *http.Client) *dispatcher {
+func newDispatcher(httpClient *http.Client, config *pb.Config) *dispatcher {
```

Comment:
The dispatcher in this case would hold a pointer to a `main.Config` struct, and serialize it into a protobuf message within the `GetConfig` implementation function.